src/m1_facilitator/manifest/Manifest.ts

```diff
@@ -226,6 +227,7 @@ export default class Manifest {
       }
       manifestConfig.metachain = Manifest.getMetachain(manifestConfig);
       manifestConfig.accounts = Manifest.getAvatarAccounts(manifestConfig);
+      manifestConfig.facilitate_tokens = Manifest.toChecksumAddress(manifestConfig);
```

Comment:
Why not `getSupportedToken` or `getSupportedTokenAddresses`
`toChecksumAddress` is very generic and can be used for any address, not just tokens.

Reply:
Done!

I also think we also should rename `facilitate_tokens` in manifest.yml to align everywhere code/config etc. We can keep it for now since we have already discussed and agreed upon. We can rename with a separate ticket.